CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 4.6.0 / 2023-07-26
+
+### ✨ New Features
+
+- Add command to really delete data in tracimcli
+- Increase size of emojis in TinyMCE
+- Upgrade pre-commit version to flacke8 6.0.0, trailing-whitespace v4.4.0, black 23.3.0, isort v5.10.1 and seed-isort-config v2.2.0
```

Comment:
I think this is not really a new feature. This is more a fixed issues, no ?

And I think it is better to just inform that we have upgraded some dependencies (and maybe add name of this dependencies)

Reply:
I placed it in fixed issues